src/Microsoft.ML.StandardTrainers/Microsoft.ML.StandardTrainers.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
+    <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
This type of change isn't going to work for the same reasons as I outlined here: [https://github.com/dotnet/machinelearning/pull/3785#discussion_r289088948](https://github.com/dotnet/machinelearning/pull/3785#discussion_r289088948).

Instead, the intrinsics classes should be moved to `Microsoft.ML.CpuMath`, which is set up to multi-target for `netstandard2.0` and `netcoreapp3.0`.

Reply:
> > Would you please help trigger the tests again?
> 
> I have retried the failing tests. If you want to do this, in Azure Pipelines, sign in, and then at the top right is a `...` button - drop that down and click "Retry".

Thanks Eric!

Reply:
> > Would you please help trigger the tests again?
> 
> I have retried the failing tests. If you want to do this, in Azure Pipelines, sign in, and then at the top right is a `...` button - drop that down and click "Retry".

Thanks Eric!

Reply:
> This type of change isn't going to work for the same reasons as I outlined here: [#3785 (comment)](https://github.com/dotnet/machinelearning/pull/3785#discussion_r289088948).
> 
> Instead, the intrinsics classes should be moved to `Microsoft.ML.CpuMath`, which is set up to multi-target for `netstandard2.0` and `netcoreapp3.0`.

Should I move the intrinsic methods in the existing `AvxIntrisics` class or should I keep them in a separate class in  `Microsoft.ML.CpuMath`?

Reply:
The separate class(es) you made sound good to me. Keeping the logic separate makes sense.